app/src/test/java/org/simple/clinic/editpatient/EditPatientInitTest.kt

```diff
@@ -17,7 +21,16 @@ class EditPatientInitTest {
   private val patientPhoneNumber = TestData.patientPhoneNumber(UUID.fromString("f204b770-83c5-4145-9ca7-c2273be2bbdc"), number = "9999999999")
   private val bangladeshNationalId = TestData.businessId(UUID.fromString("c9be5e5d-770c-4f37-a9de-2c304dfebfcd"))
   private val dateOfBirthFormat = DateTimeFormatter.ofPattern("dd/MM/yyyy", Locale.ENGLISH)
-  private val initSpec = InitSpec(EditPatientInit(patient, patientAddress, patientPhoneNumber, bangladeshNationalId))
+  private val features = Features(
+      remoteConfigService = NoOpRemoteConfigService(DefaultValueConfigReader()),
+      overrides = mapOf(
+          Feature.VillageTypeAhead to true
+      ))
```

Comment:
Yeah, as mentioned earlier, this information is unnecessary for testing business logic. Passing in the value as a `Boolean` property to `EditPatientInitTest` should simplify this a lot.